_data/home.yml

```diff
@@ -90,3 +90,43 @@ quote:
   - text: At Lyft, we've made tremendous strides in our resilience and observability since we started deploying Envoy. We're excited to be open sourcing Envoy, and the community that's growing around Envoy will help both Lyft and others adopting a microservices architecture.
     author: Peter Morelli
     role: VP Engineering, Lyft
+
+# Try to keep descriptions to about 50 words, for layout's sake
+partners:
+ - name: Datawire
+   logo: /img/logos/datawire.svg
+   website: https://www.datawire.io/
+   description: Datawire's open source toolkit empowers developers to adopt a cloud-native development workflow for Kubernetes services.
+ - name: Turbine Labs
+   logo: /img/logos/turbinelabs.svg
+   website: https://www.turbinelabs.io
+   description: Turbine Labs provides a complete traffic management solution to help teams build and operate microservices more effectively. Built on top of Envoy, Turbine Labs gives small teams the power to migrate to new infrastructure, expose new functionality, or configure resilience strategies with realtime feedback on how their services are performing.
+
+# Try to keep descriptions to about 50 words, for layout's sake.
+# logo is optional -- without it, the name will show
+
+projects:
+ - name: Ambassador
+   logo: /img/logos/ambassador.svg
+   website: https://www.getambassador.io/
+   github: https://github.com/datawire/ambassador
+   description: Ambassador is an open source Kubernetes-native API Gateway built on Envoy, designed for microservices.
+ - name: Cilium
+   logo: /img/logos/cilium.svg
+   website: https://github.com/cilium/cilium
+   github: https://github.com/cilium/cilium
+   description: Cilium is open source software for providing and transparently securing network connectivity and loadbalancing between application workloads such as application containers or processes. See https://github.com/cilium/cilium/issues/418 for Envoy support.
```

Comment:
I think we can delete the link to the GitHub issue? From reading it, it looks like they've integrated Envoy support already? cc: @tgraf

Reply:
Reference to GH issue can be removed. Envoy has been the default proxy for a while.

Reply:
k will fix! was more concerned about the project description which I cargo culted :)